server/server.js

```diff
@@ -17,8 +17,7 @@ var errorHandler = require('strong-error-handler');
 
 // required to support base models
 app.dataSource('db', {
-  connector: loopback.Memory,
-  defaultForType: 'db',
+  connector: "memory"
```

Comment:
Nitpick: please use single quotes (`'memory'`) instead of double quotes.

We use single quotes in JavaScript/TypeScript code. Double quotes are required by JSON standard, thus we use them in JSON files.